community.md

```diff
@@ -10,15 +10,15 @@ The Kotlin community is growing day by day. Join in. Take part!
 
 You can post technical questions on
 
-- [Devnet Forums on JetBrains Forums](http://devnet.jetbrains.com/community/kotlin)
-- [Stackoverflow](http://stackoverflow.com/questions/tagged/kotlin). Make sure you tag questions with **kotlin**
-- [Reddit] (http://www.reddit.com/r/Kotlin/)
+- [Kotlin forum on JetBrains DevNet](http://devnet.jetbrains.com/community/kotlin)
+- [StackOverflow](http://stackoverflow.com/questions/tagged/kotlin). Make sure you tag questions with **kotlin**
+- [Reddit](http://www.reddit.com/r/Kotlin/)
 
 ## Other mediums
 
-- [LinkedIn](https://www.linkedin.com/groups/Kotlin-Developers-7417237?gid=7417237&mostPopular=&trk=tyah&trkInfo=tarId%3A1404329340748%2Ctas%3Akotlin%2Cidx%3A2-1-6). Mostly for job related posts. 
-- [Google+](https://plus.google.com/communities/104597899765146112928)
 - [Twitter](http://twitter.com/project_kotlin)
-
+- [LinkedIn](https://www.linkedin.com/groups/Kotlin-Developers-7417237?gid=7417237&mostPopular=&trk=tyah&trkInfo=tarId%3A1404329340748%2Ctas%3Akotlin%2Cidx%3A2-1-6). Mostly for job related posts.
+- [Google+](https://plus.google.com/communities/104597899765146112928)
+- #kotlin channel on the [freenode.net](http://freenode.net) IRC network
```

Comment:
Are we good with promoting this now? I had it on here initially but then removed it.